### PR TITLE
Create anonymous report form. First attempt at it.

### PR DESCRIPTION
**Pull request by ohhelloana**
I think this will work. Fingers crossed.

### CODE REVIEW

File: src/_includes/partials/anonymous-contact-form.njk [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  </h1>
+</header>
+
+<article class="bg copy page__content">
+  {{ content | markdown | safe }}
+
+  <br>
+
+  <form name="coc-report" method="POST" data-netlify="true" action="/success-message">
+    <p>
```
Comment (oliverturner) [inline src/_includes/partials/anonymous-contact-form.njk:16]: Probably worth adding the bot-trapping [honeypot input](https://www.netlify.com/docs/form-handling/#spam-filtering) too

File: src/about-us.njk [hunk 2]
```diff
@@ -16,7 +16,7 @@ content: |
 
   ## Team
 
-  London CSS has been curated by [Ana Rodriguez](https://twitter.com/ohhelloana) and [Oliver Turner](https://twitter.com/oliverturner) since late 2018. 
+  London CSS has been curated by [Ana Rodrigues](https://twitter.com/ohhelloana) and [Oliver Turner](https://twitter.com/oliverturner) since late 2018.
```
Comment (oliverturner) [inline src/about-us.njk:19]: Sorry about that!

**Review by oliverturner — CHANGES_REQUESTED** [review]
This is great! 

One small ask: since this project uses Yarn, would you mind deleting  `package-lock.json`? 

As soon as that's gone I'll be delighted to approve :)

File: src/about-us.njk (hunk 2 above)
Comment (ohhelloana) [reply src/about-us.njk:19]: no worries! i really don't mind :)

File: src/_includes/partials/anonymous-contact-form.njk [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+  {{ content | markdown | safe }}
+
+  <br>
+
+  <form name="coc-report" method="POST" netlify-honeypot="bot-field" data-netlify="true" action="/success-message" >
+    <p>
+      <label>Message:</label>
+      <br> 
+      <textarea name="report" cols="40" rows="10" id="message" required="true"></textarea>
+    </p>
```
Comment (oliverturner) [inline src/_includes/partials/anonymous-contact-form.njk:20]: So we just need to add an input whose name matches the value of the `data-honeypot` attribute and a `.hidden` class to the CSS... and we're good to go 👍

```html
<p class="hidden">
  <label>Don’t fill this out if you're human: 
    <input name="bot-field" />
  </label>
</p>
```

**Comment by oliverturner** [comment]
Closing this as superseded by #10

**Pull request closed without merging**